packages/manager/apps/hub/src/index.js

```diff
@@ -42,4 +45,7 @@ angular
       $transitions.onBefore({ to: 'app.**' }, () => $translate.refresh());
     },
   )
+  .run(($translate) => {
+    moment.locale(head($translate.use().split('_')));
```

Comment:
Would it be better to get this information from me.language?

Reply:
`me.language` is not the same as the manager language :)

Reply:
Yeah... Sorry